packages/boot/test/bootstrapTests/updateUpgradedVaultParams.test.ts

```diff
@@ -5,40 +5,65 @@
  * objects from the contract held by the governor are gone, then try to change
  * param again, to show that the bug is fixed.
  */
-import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
 import type { TestFn } from 'ava';
-import { makeAgoricNamesRemotesFromFakeStorage } from '@agoric/vats/tools/board-utils.js';
-import { Fail } from '@endo/errors';
 
-import { makeSwingsetTestKit } from '../../tools/supports.js';
+import { updateVaultManagerParams } from '@aglocal/boot/test/tools/changeVaultParams.js';
 import {
   makeGovernanceDriver,
   makeWalletFactoryDriver,
-} from '../../tools/drivers.js';
-import { updateVaultManagerParams } from '../tools/changeVaultParams.js';
+} from '@aglocal/boot/tools/drivers.js';
+import { makeMockBridgeKit } from '@agoric/cosmic-swingset/tools/test-bridge-utils.ts';
+import { makeCosmicSwingsetTestKit } from '@agoric/cosmic-swingset/tools/test-kit.js';
+import { unmarshalFromVstorage } from '@agoric/internal/src/marshal.js';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import {
+  boardSlottingMarshaller,
+  makeAgoricNamesRemotesFromFakeStorage,
+  slotToBoardRemote,
+} from '@agoric/vats/tools/board-utils.js';
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { Fail } from '@endo/errors';
+
+const { fromCapData } = boardSlottingMarshaller(slotToBoardRemote);
 
-const makeDefaultTestContext = async t => {
+const makeDefaultTestContext = async () => {
   console.time('DefaultTestContext');
-  const swingsetTestKit = await makeSwingsetTestKit(t.log);
 
-  const { runUtils, storage } = swingsetTestKit;
+  const storage = makeFakeStorageKit('bootstrapTests');
+  const swingsetTestKit = await makeCosmicSwingsetTestKit({
+    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
+    mockBridgeReceiver: makeMockBridgeKit({ storageKit: storage }),
+  });
+
+  const { EV, queueAndRun } = swingsetTestKit;
+
+  const readLatestEntryFromStorage = (path: string) => {
```

Comment:
This seems to be defined a lot; maybe there should be a single helper somewhere in `packages/boot/`.